src/vdaf/xof.rs

```diff
@@ -216,7 +216,7 @@ impl Xof<16> for XofShake128 {
     }
 }
 
-/// The key stream produced by the cSHAKE128 XOF.
+/// The seed stream produced by the SHAKE128.
```

Comment:
This doesn't read right, we should do one of the following
```suggestion
/// The seed stream produced by the SHAKE128 XOF.
```
```suggestion
/// The seed stream produced by SHAKE128.
```